src/test/kotlin/uk/gov/justice/digital/hmpps/whereabouts/services/OffenderEventServiceTest.kt

```diff
@@ -114,8 +104,8 @@ class OffenderEventServiceTest {
         return event
     }
 
-    private fun convertToDto(oe: OffenderEvent): OffenderEventDto {
-        return OffenderEventDto.builder()
+    private fun convertToDto(oe: OffenderEvent): OffenderEventDto =
```

Comment:
with kotlin you don't need `return` if you don't want one :grin:

Reply:
nice!